wazo_router_confd/tests/test_api_carrier_trunks_auth.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
```

Comment:
It's time to bump the year in your snippet ;) (this is not a blocker)

Reply:
What is the policy of Copyright year in Wazo? Shall we bump it in all files? Just the files we modify?

Reply:
We add the current year when we modify the file. And we only include the first and last years.

For example `2014-2020` not `2014, 2018-2020`

There a git hook here https://github.com/wazo-platform/xivo-tools/blob/master/dev-tools/git-hooks/copyright-check that some of us use